src/torrent.cpp

```diff
@@ -11821,7 +11830,7 @@ namespace {
 		// if we don't have any metadata, stop here
 
 		st->queue_position = queue_position();
-		st->need_save_resume = need_save_resume_data();
+		st->need_save_resume = bool(m_need_save_resume_data);
```

Comment:
So how about add yet another field to represent it in details?

Reply:
that would be an ABI break. I could do that in `master`

Reply:
> that would be an ABI break. I could do that in `master`

OK. Just don't delay doing it, otherwise it may be forgotten.